### PR TITLE
Alpha version

### PR DESCRIPTION
**Pull request by brechtvdv**

### CODE REVIEW

File: lib/StreamIterator.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 var util = require('util'),
     EventEmitter = require('events').EventEmitter;
 
+EventEmitter.prototype._maxListeners = 1000;
```
Comment (pietercolpaert) [inline lib/StreamIterator.js:4]: Ehm... If you need to do this... there's something really wrong ;)

File: lib/arrdep2connections.js [hunk 2]
```diff
@@ -14,8 +13,7 @@ function ArrDep2Connections (arrivalsStream, options) {
 }
 
 ArrDep2Connections.prototype._transform = function (departure, encoding, done) {
-  departure = JSON.parse(departure.value);
-  departure["departureTime"] = new Date(departure["departureTime"]);
+  departure["gtfs:departureTime"] = departure["date"] + 'T' + departure["gtfs:departureTime"];
```
Comment (pietercolpaert) [inline lib/arrdep2connections.js:16]: then it's not a `gtfs:departureTime` anymore ;-) then it's a `lc:departureTime` though

File: lib/arrdep2connections.js [hunk 3]
```diff
@@ -23,15 +21,20 @@ ArrDep2Connections.prototype._transform = function (departure, encoding, done) {
       var processNextArrival = function () {
         self._getNextArrival(function (nextArrival) {
           if (nextArrival) {
-            setImmediate(function () {
-              self._processArrival(departure, nextArrival, processNextArrival, done);
-            });
+            if (departure["gtfs:stopSequence"] != departure["maxStopSequence"]) {
+              setImmediate(function () {
```
Comment (pietercolpaert) [inline lib/arrdep2connections.js]: Ufff... You had some hard async thinking going on here :D

File: lib/StreamIterator.js (hunk 1 above)
Comment (brechtvdv) [inline lib/StreamIterator.js:4]: Some legacy of my debugging :p

**Pull request merged**